puppet/modules/web/manifests/vhost/yum.pp

```diff
@@ -27,11 +29,21 @@
     },
   ]
 
+  if $setup_receiver {
+    secure_ssh::receiver_setup { $user:
+      user           => $user,
+      foreman_search => 'host ~ node*.jenkins.osuosl.theforeman.org and (name = external_ip4 or name = external_ip6)',
```

Comment:
Btw, I think we should pull this up to the profile level and unify with the web user since that's the same setup. I'll take a stab at looking how we reuse those keys and credentials but IMHO that doesn't have to block deployment.